core/state/snapshot/generate.go

```diff
@@ -250,7 +256,10 @@ func (dl *diskLayer) generate(stats *generatorStats) {
 		accMarker = nil
 	}
 	if err := accIt.Err; err != nil {
-		log.Crit("Generator failed to iterate account trie", "root", dl.root, "err", err)
+		log.Error("Generator failed to iterate account trie", "root", dl.root, "err", err)
```

Comment:
You could add the `it.Path` there too, it's good info to have

Reply:
It's in the error: `ERROR[10-05|11:06:01.627] Generator failed to iterate storage trie accroot="e3712f…b9b1fd" acchash="291849…881040" stroot="ddefcd…f2bc67" err="missing trie node 18a0f4d79cff4459642dd7604f303886ad9d77c30cf3d7d7cedb3a693ab6d371 (path 04)"`

Reply:
I can add it explicitly too if you want